samples/gazell/gzll_ack_payload_device/README.rst.orig

```diff
@@ -0,0 +1,32 @@
+.. _gzll_ack_payload_device:
```

Comment:
This file seems like a remainder from a conflict resolution in git. Please remove this file.

Reply:
Thank you for pointing this out! I will actually try to remove these `.rst.orig` files from this commit if possible, as these were created from solving my merge conflicts I believe.